rust/src/plutus.rs

```diff
@@ -616,7 +616,207 @@ impl Strings {
 
 
 
+// json
 
+#[wasm_bindgen]
```

Comment:
I'll document this better/fully once we implement the rest of the BasicConversions format